apps/ledger-live-desktop/src/renderer/screens/platform/v2/Catalog/Card/Full.tsx

```diff
@@ -58,7 +57,9 @@ export function FullCard(props: PropsRaw) {
               </Text>
             )}
           </Flex>
-          <Subtitle>{subtitle}</Subtitle>
+          <Subtitle>
+            {(manifest.content.subtitle && manifest.content.subtitle[language]) || hostname}
+          </Subtitle>
```

Comment:
You probably need to implement a fallback to english too because we don't have all languages in all manifests and it's usually preferable to fallback on the english version

Edit: just saw the `translateContent` below and it's doing the fallback so you should try to use it